pkg/tracing/tracing.go

```diff
@@ -512,6 +517,17 @@ func (t *Tracer) newSpanWithStart(ctx context.Context, name string, startAt time
 	return &Span{Span: span, ctx: ctx, tracer: t}
 }
 
+func (t *Tracer) newSpanWithCDN(ctx context.Context, name string, req *http.Request) *Span {
```

Comment:
the function name is `SpanWithCDN`, but the implementation only supports cloudflare, which does not align with the implementation of `enableCDN` (currently, `enableCDN` only supports cloudflare too, but it is easy to see it will be extended to support other CDNs) .